x11-libs/libxaw/libxaw-1.0.13.recipe

```diff
@@ -0,0 +1,116 @@
+SUMMARY="X Athena Widget library"
+DESCRIPTION="X Athena Widgets, or Xaw, is a widget set based on the X Toolkit \
+Intrinsics (Xt) Library. Xaw & Xt form together what is known as the X Toolkit."
+HOMEPAGE="https://www.x.org/releases/individual/lib/"
+COPYRIGHT="1985-1990, 1994, 1998 The Open Group
+	2003-2004 Roland Mainz
+	1998-1999 The XFree86 Project, Inc.
+	1991 OMRON Corporation
+	1985-1998 Digital Equipment Corporation
+	1989 Prentice Hall
+	1985-1989, 1991, 1994 X Consortium"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://www.x.org/releases/individual/lib/libXaw-$portVersion.tar.bz2"
+CHECKSUM_SHA256="8ef8067312571292ccc2bbe94c41109dcf022ea5a4ec71656a83d8cce9edb0cd"
+SOURCE_DIR="libXaw-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion6="6.0.1"
+libVersion7="7.0.0"
+
+libVersionCompat6="$libVersion6 compat >= 6"
+libVersionCompat7="$libVersion7 compat >= 7"
+
+PROVIDES="
+	libxaw$secondaryArchSuffix = $portVersion
+	lib:libxaw$secondaryArchSuffix
```

Comment:
Do you think we need a compat version for this? If a port has been built with Xaw, HaikuPorter will complain that `lib:libxaw` is missing from `REQUIRES`. But then, a definition like this `lib:libxaw >= 6` won't work to get Xaw6